src/altinn-studio-cli/Upgrade/Frontend/Fev3Tov4/FrontendUpgrade/FrontendUpgrade.cs

```diff
@@ -122,6 +129,11 @@ public static Command GetUpgradeCommand(Option<string> projectFolderOption)
                     returnCode = await LayoutSetUpgrade(uiFolder, layoutSetName, applicationMetadataFile);
                 }
 
+                if (!skipCustomReceiptUpgrade && returnCode == 0)
```

Comment:
Noticed that the CustomReceiptUpgrade needs to run after the LayoutSetUpgrader. Would that potentially cause problems if layoutsets are not upgraded by providing `--skip-layout-upgrade`? This is IMO the true definition of an edge-case but might be something to consider?

Reply:
This is checked later here:
https://github.com/Altinn/altinn-studio-cli/blob/6d25725c86c6ceb9c3422ff47b2f95d4391fb879/src/altinn-studio-cli/Upgrade/Frontend/Fev3Tov4/FrontendUpgrade/FrontendUpgrade.cs#L236-L240